recipes/pymorphy2-dicts-ru/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pymorphy2-dicts-ru" %}
+{% set version = "2.4.404381.4453942" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e636a633de4d37b32939fd9f97c1e65b109a5a14f6c87a83d1f44f505020bd4c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```

Comment:
Is setuptools really a runtime dep?

Reply:
@ scopatz you're right, forgot to remove it from the skeleton. and there were a few other unnecessary ones i just removed also.

Reply:
Sweet thanks!